reconcile.py

```diff
@@ -57,6 +54,23 @@ def fill_email_ids(all_clusters, config):
         pbar.update()
 
 
+def apply_non_portal_reimbursements(config, trackings_to_costs_map: Dict[Tuple[str], Tuple[str,float]],
+                                    po_to_cost_map: Dict[str, float]) -> None:
+  non_portal_reimbursements = NonPortalReimbursements(config)
+  for tracking_tuple in non_portal_reimbursements.trackings_to_costs.keys():
```

Comment:
Why not just use set intersection?  Then you can get the entire list of trackings that are incorrectly included in two places and output them all.

Reply:
sure we can do that; it shouldn't matter either way